modules/ROOT/pages/configure-multiple-credential-providers.adoc

```diff
@@ -49,29 +48,31 @@ If you have external client providers such as PingFederate configured for your e
 
 *Best Practice*
 
-To secure your APIs by leveraging the multiple client providers feature, create one external client provider per environment. Assign the existing client provider to your production environment and add the new providers to other environments, such as sandbox, and QA. 
+To secure your APIs, create one external client provider per environment. Assign the existing client provider to your production environment and add the new providers to QA and to other environments, such as the sandbox. However, check the possible impact on managed APIs and their consumers. 
 
-However, before changing any existing client providers, ensure that you check the impact on managed APIs and their consumers. Before you implement multiple client providers, see the <<guidelines,guidelines>>.
+Avoid using the same IdP in production and nonproduction environments. You can use the same IdP in multiple production environments or in multiple nonproduction environments.
 
+Before implementing multiple client providers, see the <<guidelines,guidelines>>.
 
 [[guidelines]]
 == Guidelines for Implementing Multiple Client Providers
 
 Before you implement multiple client providers in Anypoint Platform, carefully review and understand the following points:
 
-* When you change the client provider for an existing API instance, ensure that the API:
-** Has no contracts
-** Has no client provider-related policies applied, such as PingFederate and OpenID policies
+* You cannot change a client provider's existing API instance when that API:
+** Has contracts
+** Has client provider-related policies applied, such as PingFederate and OpenID policies
```

Comment:
This particular bullet has a small exception. A user can change a client provider's existing API instance when the provider type doesn't change.
A user can change the IDP of an API with PingFederate-related policies applied can be swapped between `PingForDev` and `PingForProd` because both IDPs are of the type `PingFederate`.